projects/storefrontlib/cms-structure/page/slot/page-slot.component.spec.ts

```diff
@@ -138,8 +138,8 @@ describe('PageSlotComponent', () => {
   });
 
   describe('position', () => {
-    it('should return undefined for position', () => {
-      expect(pageSlotComponent.position).toBeUndefined();
+    it('should return empty string for position', () => {
+      expect(pageSlotComponent.position).toEqual('');
```

Comment:
Was there an alternative to changing this behaviour?

Reply:
revert back